api/data/NMBS/vehicleinformation.php

```diff
@@ -401,4 +431,14 @@ private static function getServerDataByUrl($url)
 
         return $result;
     }
+
+    private static function getURLForOccupancy($occupancy) {
```

Comment:
To be strict, it's a URI, not a URL. A URL is a locator of a document, a URI is an identifier.